v2/internal/controllers/crd_aks_managedcluster_20230102preview_test.go

```diff
@@ -162,3 +169,53 @@ func AKS_ManagedCluster_Kubeconfig_20230102Preview_Secrets(tc *testcommon.KubePe
 	tc.PatchResourceAndWait(old, cluster)
 	tc.ExpectSecretHasKeys(secret, "admin", "user")
 }
+
+func AKS_ManagedCluster_TrustedAccessRoleBinding_20230102Preview_CRUD(
+	tc *testcommon.KubePerTestContext,
+	resourceGroup *v1api20200601.ResourceGroup,
+	cluster *aks.ManagedCluster,
+) {
+
+	// Create a storage account and key vault to use for the workspace
+	sa := newStorageAccount(tc, resourceGroup)
+	tc.CreateResourceAndWait(sa)
+
+	kv := newVault("kv", tc, resourceGroup)
+	tc.CreateResourceAndWait(kv)
+
+	// Create workspace
+	workspace := newWorkspace(
+		tc,
+		testcommon.AsOwner(resourceGroup),
+		sa,
+		kv,
+		resourceGroup.Spec.Location,
+	)
+	tc.CreateResourceAndWait(workspace)
+
+	roleBinding := &aks.TrustedAccessRoleBinding{
+		ObjectMeta: tc.MakeObjectMetaWithName("tarb"),
+		Spec: aks.ManagedClusters_TrustedAccessRoleBinding_Spec{
+			Owner: testcommon.AsOwner(cluster),
+			Roles: []string{
+				workspace.GetType() + "/mlworkload",
```

Comment:
A comment here would be good on what the actual value looks like.

Reply:
Done.